testsuite/integration-arquillian/servers/app-server/tomcat/pom.xml

```diff
@@ -99,29 +108,92 @@
                                 </configuration>
                             </execution>
                             <execution>
-                                <id>libs-for-tomcat</id>
+                                <id>libs-for-tomcat-7</id>
                                 <phase>generate-test-resources</phase>
                                 <goals>
                                     <goal>copy</goal>
                                 </goals>
                                 <configuration>
+                                    <skip>${skip.dependencies.for.tomcat7}</skip>
                                     <artifactItems>
                                         <artifactItem>
                                             <groupId>org.jboss.resteasy</groupId>
-                                            <artifactId>resteasy-client</artifactId>
+                                            <artifactId>resteasy-jaxrs</artifactId>
+                                            <version>${tomcat.resteasy.version}</version>
                                         </artifactItem>
+                                        <artifactItem>
+                                            <groupId>org.scannotation</groupId>
+                                            <artifactId>scannotation</artifactId>
+                                            <version>1.0.3</version>
```

Comment:
Could we extract the versions to parent pom (by using some BOM or by declaring versions in properties). This should make it a bit easier during the upgrades.

The same comment applies to other dependencies in other files.

Reply:
I am not sure we want to extract those. These are just dependencies for tomcat, we are not using it anywhere in testsuite except for this place. Even if we want to use those dependencies we may need some other version.

Reply:
Fair point. Is there any BOM available that would include them? I'm just a little bit concerned how do upgrade those artifacts in the future and make sure they work together? If there's no other option, please keep it as it.

Reply:
I am not sure, I didn't find anything. I used version from keycloak/pom.xml where it is possible and extracted others to tomcat parent pom properties. Are you ok with this solution?